detection_rules/devtools.py

```diff
@@ -916,15 +917,20 @@ def rule_survey(ctx: click.Context, query, date_range, dump_file, hide_zero_coun
     click.echo(f'Saving detailed dump to: {dump_file}')
```

Comment:
```suggestion
    click.echo(f'Will save detailed dump to: {dump_file}')
```

I accidentally deleted too much. The point was that the output to save the dump occurs at the very beginning (and then an extended amount of time passes). It makes more sense to either leave as it and move it to right before actually saving the dump, or slightly change the message to say that it `Will save`.

Reply:
it intentionally does this because it goes to a pager for long text, so this notifies the user before hand that it will be saving to that location after the pager